openapi.yaml

```
@@ -360,6 +360,34 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Error"
+  /statistics/snapshot:
```

This would probably be more Restish:
```suggestion
  /statistics/snapshot/download:
```

---

fixed in the ticket